airflow/example_dags/tutorial.py

```diff
@@ -100,6 +98,11 @@
     ![img](http://montcs.bloomu.edu/~bobmon/Semesters/2012-01/491/import%20soul.png)
     """
     )
+
```

Comment:
can we use https://user-images.githubusercontent.com/8811558/113521151-7d4d1e00-958f-11eb-9571-10cbc9223193.png for Task Docs ?

since it shows the image on new line.

That image is from my PR: https://github.com/apache/airflow/pull/15191

Reply:
![](https://user-images.githubusercontent.com/8811558/113521151-7d4d1e00-958f-11eb-9571-10cbc9223193.png)